### DISCUSSION TITLE
YoutubeSeriesDownloader - Cookies file not found

### DISCUSSION THREAD

**Discussion by solidsnakex37** [RandomNinjaAtk/arr-scripts#175] [Q&A] [answered]
I just set up this script last week and testing things out, it seems that the script finds the series but it's not downloading anything, I wanted to check if I might be missing something? 

The logs say:

2024-02-04 00:39:14 :: YoutubeSeriesDownloader :: 2.0 :: Script starting...
2024-02-04 00:39:14 :: YoutubeSeriesDownloader :: 2.0 :: Cookies File Not Found!
2024-02-04 00:39:15 :: YoutubeSeriesDownloader :: 2.0 :: 2/3 :: 1/1 :: Project Fear (2023) :: S3E1 :: ERROR :: No Download URL found, skipping
2024-02-04 00:39:15 :: YoutubeSeriesDownloader :: 2.0 :: Script sleeping for 1h...

Few questions: 

What is the cookies file? Is this something I need to setup? I didn't see this in the documentation, if I look at the script it is looking for /config/cookies.txt but I have no idea what it's expecting to be in there

**Comment by RandomNinjaAtk** [comment, marked as answer]
https://github.com/RandomNinjaAtk/arr-scripts/issues/56#issuecomment-1674828395
https://github.com/RandomNinjaAtk/docker-sonarr-extended/issues/21

**Reply by solidsnakex37** [reply]
Thank you, that's why I couldn't find it, didn't realize the cookies.txt was a yt-dlp dependency 

I am still not understanding the cookie.txt thing, can you explain how your script it looking for the download URL, and why it might return the error "No Download URL found" ? 

Using the cookies.txt seemed to have made no difference, so perhaps that wasn't an issue and it's just an info message that one isn't being used

**Reply by RandomNinjaAtk** [reply]
Read through the second link
